_posts/2020-10-04-newpipe-0.20.0.md

```diff
@@ -1,154 +1,176 @@
 
-Thanks to [@wb9688](https://github.com/wb9688), you can open a video in the local playlist tab (and potentially other locations), whereas earlier, the app would crash. And thanks to [@BoFFire](https://github.com/BoFFire), if you open the app in the Taqbaylit language, its called `Taqbaylit` now and not `Tamaziɣt Taqbaylit`.
+Instead of causing a crash, thanks to [@wb9688](https://github.com/wb9688), you can open a video in the local playlist tab (and potentially other locations). Thanks to [@BoFFire](https://github.com/BoFFire), if you open the app in the Taqbaylit language, it goes by that name, and not `Tamaziɣt Taqbaylit`.
 
-The license pop-up doesn't disappear when you change orientation now. This was introduced by [@nmurali94](https://github.com/nmurali94). And [@budde92](https://github.com/budde25) ensured that you can now delete a file you just downloaded and then try to re-download it without the app crashing. We are not sure why you'd want to do this, but now you can do so without any worries.
+The license pop-up now lacks the ability to just disappear when you change orientation. This was fixed by [@nmurali94](https://github.com/nmurali94).
+[@budde92](https://github.com/budde25) clicked enough keyboard to where you can now delete a file you just downloaded _and_ re-download it _without_ the app crashing. In what is a conundrum, we are not sure why you'd want to before now, but do.
 
-[@mhmdanas](https://github.com/mhmdanas) changed a code call because its origin was redundant. Yay for less redundancy. When auto-queue is enabled, Newpipe wants to add a stream to the queue the moment you open a video. But if there are no video suggestions on that page, it causes a crash. This has been fixed by [@wb9688](https://github.com/wb9688).
+[@mhmdanas](https://github.com/mhmdanas) changed a code call because its origin was redundant. Less redundancy good. When auto-queueing takes place, Newpipe wants to add a stream to the queue the moment you open a video. If no video suggestions were to be found on that page, that caused a crash. And then [@wb9688](https://github.com/wb9688) put an end to such shenanigans.
 
 ## Development
 
-Development. Where all the magic which keeps this app running happens. It saw some improvements in this release: (Warning: Nerd talk ahead)
+Development. The nerd magic sourced from the purest of autism. Embrace yourself:
 
 - The extractor will ignore OTF streams now, since they don't work with progressive HTTP, and the extractor can't work with DASH just yet. This has been a major point of inconvenience for a lot of users (and us, too!) so we're eager to mitigate the problem until the proper fix sometime later. ([@wb9688](https://github.com/wb9688))
```

Comment:
```suggestion
- The extractor will ignore OTF streams now, since they don't work with progressive HTTP, and the extractor can't work with DASH just yet. This has been a major inconvenience for a lot of users and developers alike, so we're eager to mitigate the problem until the proper fix can be landed sometime later. ([@wb9688](https://github.com/wb9688))
```